pkg/cli/alpha/internal/update/update.go

```diff
@@ -0,0 +1,528 @@
+/*
+Copyright 2025 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package update
+
+import (
+	"errors"
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+	"os/exec"
+	"runtime"
+	"strings"
+	"time"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/afero"
+	"golang.org/x/mod/semver"
+
+	"sigs.k8s.io/kubebuilder/v4/pkg/cli/alpha/internal/common"
+	"sigs.k8s.io/kubebuilder/v4/pkg/config/store"
+	"sigs.k8s.io/kubebuilder/v4/pkg/plugin/util"
+)
+
+const releaseURL = "https://github.com/kubernetes-sigs/kubebuilder/releases/download/%s/kubebuilder_%s_%s"
```

Comment:
Previously, this value was defined inside a struct, but since it’s a constant that doesn't change, it makes more sense to define it as a global const.

By doing this, we can reuse it consistently both in the validation logic (to check if the URL exists) and in the function that downloads the binary. It also helps avoid duplication and makes the intent clearer.